src/App.tsx

```diff
@@ -19,17 +19,21 @@ const globalStyles = css`
 `;
 
 const App: React.VFC = () => (
-  <ErrorBoundary>
-    <ContextProviders>
-      <Router>
-        <Route path={routes.login} component={Login} />
-        <GQLWrapper>
-          <Global styles={globalStyles} />
-          <Content />
-        </GQLWrapper>
-      </Router>
-    </ContextProviders>
-  </ErrorBoundary>
+  <>
+    <Global styles={globalStyles} />
```

Comment:
Global should be within error boundary. Since the error boundary will catch all of the errors in the app.